package.json

```
@@ -47,12 +47,11 @@
   },
   "dependencies": {
     "async": "2.6.1",
-    "request": "~2.88.0",
     "underscore": "1.12.1",
     "cheerio": "1.0.0-rc.2",
     "xmldom": "0.1.27",
     "logops": "2.1.0",
-    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "iotagent-node-lib": "git://github.com/jason-fox/iotagent-node-lib.git#feature/request-shim",
```

@jason-fox As described in https://github.com/telefonicaid/iotagent-node-lib/pull/1119#issuecomment-1000483835, this need to be addressed (reverted) as first step of the process. 

```suggestion
    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
```